src/lib/change-handlers.ts

```diff
@@ -145,7 +146,8 @@ export const handleTextChange = (
 
 export const handleNodeChange = (
   suggestion: NodeChange | NodeAttrChange,
-  state: ManuscriptEditorState
+  state: ManuscriptEditorState,
+  getFiles?: FileAttachment[]
```

Comment:
the name suggest a function, but you receive an array. Please consider renaming to just "files" or similar.

Reply:
Done